APIWrapper/Enums/eSubjectType.cs

```diff
@@ -0,0 +1,14 @@
+
+namespace APIWrapper.Enums
+{
+    public enum eSubjectType
+    {
+        None = 0,
+
+        Compulsory = 1,
+
+        Optional = 2,
```

Comment:
If you have CompulsoryElective, this should be Elective...

Reply:
ok